### ISSUE TITLE
Unable to install shadcn/ui

### ISSUE THREAD

**Issue by shmuli9** [jacobparis/underkill-stack#11]
Not sure this is an issue for this repo necessarily, but shadcn doesn't support tailwind 4 alpha as far as I can work out. Bit of a shame really

**Comment by jacobparis** [comment]
Yeah it's annoying, their CLI just bails with no way out if you try to initialize a project without a perfect happy path tailwind config file set up. They should at least let you resolve it manually
